oshi-core/src/main/java/oshi/hardware/platform/windows/WindowsHardwareAbstractionLayer.java

```diff
@@ -120,4 +123,9 @@ public Sensors getSensors() {
     public SoundCard[] getSoundCards() {
         return WindowsSoundCard.getSoundCards().toArray(new SoundCard[0]);
     }
+
+    @Override
+    public User getUser() {
+        return new WindowsUser("",0,"","");
```

Comment:
I don't think I'm going to approve this PR for Users until it's implemented on Windows.   You may want to separate out the `User` portion from the other portions as this has a ways to go.